test/python/circuit/test_uc.py

```diff
@@ -85,6 +85,22 @@ def test_global_phase_ucg(self):
 
         self.assertTrue(np.allclose(unitary_desired, unitary))
 
+    def test_inverse_ucg(self):
+        """ "Test inverse function of uniformly controlled gates"""
+        gates = [random_unitary(2).data for _ in range(2**2)]
```

Comment:
I think it's fine if you just hardcode one or two matrices, we try to avoid any random processes in the tests if possible. Or you could set a seed to ensure it is always the same matrix.

Reply:
Thanks @Cryoris .

Reply:
Thanks for the guideline, @Cryoris .